qiskit/optimization/problems/quadratic_program.py

```diff
@@ -199,6 +337,66 @@ def continuous_var(self, lowerbound: Union[float, int] = 0,
         """
         return self._add_variable(lowerbound, upperbound, Variable.Type.CONTINUOUS, name)
 
+    def continuous_var_dict(self,
+                            lowerbound: Union[float, int] = 0,
+                            upperbound: Union[float, int] = INFINITY,
+                            name: Optional[str] = None,
+                            formatter: Optional[str] = None,
+                            keys: Union[int, Sequence] = 1) -> Dict[str, Variable]:
+        """
+        Uses 'var_dict' to construct a dictionary of continuous variables
+
+        Args:
+            lowerbound: The lower bound of the variable(s).
+            upperbound: The upper bound of the variable(s).
+            name: The name(s) of the variable(s).
+            formatter: The format used to name/index the variable(s).
+            keys: If keys: int, it is interpreted as the number of variables to construct.
+                  Otherwise, the elements of the sequence are converted to strings via 'str' and
+                  passed to the formatter.
+
+        Returns:
+            A dictionary mapping the variable names to variable instances.
+
+        Raises:
+            QiskitOptimizationError: if the variable name is already taken.
+            QiskitOptimizationError: if less than one variable instantiation is attempted.
+            QiskitOptimizationError: if the formatter has more than one substitution or a
+                                     nested substitution.
+        """
+        return self.var_dict(lowerbound, upperbound, Variable.Type.CONTINUOUS, name, formatter,
+                             keys)
+
+    def continuous_var_list(self,
+                            lowerbound: Union[float, int] = 0,
+                            upperbound: Union[float, int] = INFINITY,
+                            name: Optional[str] = None,
+                            formatter: Optional[str] = None,
+                            keys: Union[int, Sequence] = 1) -> List[Variable]:
+        """
+        Uses 'var_list' to construct a list of continuous variables
+
+        Args:
+            lowerbound: The lower bound of the variable(s).
+            upperbound: The upper bound of the variable(s).
+            name: The name(s) of the variable(s).
+            formatter: The format used to name/index the variable(s).
```

Comment:
`key_format` might be better name to align with docplex.
http://ibmdecisionoptimization.github.io/docplex-doc/mp/docplex.mp.model.html#docplex.mp.model.Model.continuous_var_list